monkey/monkey_island/cc/services/attack/attack_report.py

```diff
@@ -25,7 +26,13 @@ def generate_new_report():
         Generates new report based on telemetries, replaces old report in db with new one.
         :return: Report object
         """
-        report = {'techniques': {}, 'latest_telem_time': AttackReportService.get_latest_attack_telem_time(), 'name': REPORT_NAME}
+        report =\
+            {
+                'techniques': {},
+                'meta': {'latest_monkey_modifytime': NodeService.get_latest_modified_monkey()[0]['modifytime']},
```

Comment:
Code duplication between here and line 53. Perhaps it's a good opportunity to move this function + a wrapper for getting just the time to the models module and implement using mongoengine instead? :)